core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogBomScanner.java

```diff
@@ -137,9 +153,19 @@ public void removedBundle(Bundle bundle, BundleEvent bundleEvent, Iterable<? ext
             }
         }
 
+        private void removeFromCatalog(CatalogItem<?, ?> item) {
+            try {
+                getManagementContext().getCatalog().deleteCatalogItem(item.getSymbolicName(), item.getVersion());
+            } catch (Exception e) {
+                LOG.warn("Failed to remove {} {} from catalog as {}", new String[] {
+                    item.getSymbolicName(), item.getVersion(), e.getMessage()
```

Comment:
Why not pass the exception as a throwable argument to log, instead of including the message only. Could mask an underlying problem (for example NPE will have an empty message).